bouncer/views.py

```diff
@@ -96,6 +92,54 @@ def index(request):
         location=request.registry.settings["hypothesis_url"])
 
 
+@view.view_config(renderer="bouncer:templates/annotation.html.jinja2",
```

Comment:
Yes, although the content in that template (visual & meta tags) specifically refers to a _single_ annotation. I think we'll probably want to revise the interstitial page a bit for this endpoint, but my plan was to solicit thoughts on that once the endpoint is usable.

OK if I defer this for now?